assets/source/setup-guide/app/components/TermsAndConditionsModal.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import {
+	Modal,
+	__experimentalText as Text, // eslint-disable-line @wordpress/no-unsafe-wp-apis --- _experimentalText unlikely to change/disappear and also used by WC Core
+} from '@wordpress/components';
+import { Link } from '@woocommerce/components';
```

Comment:
Not blocker, but to make the bot happy you can place this line above of the `@wordpress/components` item.

Reply:
Done.